src/Symfony/Component/HttpKernel/Profiler/ProfilerStorageInterface.php

```diff
@@ -35,6 +36,8 @@ interface ProfilerStorageInterface
      */
     public function find(?string $ip, ?string $url, ?int $limit, ?string $method, int $start = null, int $end = null): array;
 
+    public function findByType(?string $virtualType, ?string $ip, ?string $url, ?int $limit, ?string $method, int $start = null, int $end = null, string $statusCode = null): array;
```

Comment:
why do we have to add this method? why can't we filter its result instead?

Reply:
Done.